README.md

```diff
@@ -78,19 +77,17 @@ sudo make install
 
 #### Get dependencies
 
-- Get the Qt online installer from: https://www.qt.io/download-open-source
-During installation, choose a Qt 5.x with Mingw32 32-bit toolchain and CMake.
+- Get the Qt online installer from https://www.qt.io/download-open-source. During installation, choose a Qt 5.x with Mingw32 32-bit toolchain and CMake.
```

Comment:
a) is there a danger that adding a full-stop at the end of the URL means that people might try and include the full-stop in the URL itself?
b) whatever grammar-tweaks you make here, you should probably also duplicate in the Mac OS section?

Reply:
> a) is there a danger that adding a full-stop at the end of the URL means that people might try and include the full-stop in the URL itself?

Yes, but it should be fairly obvious from the fact it is at the end of a sentence that it's only there as punctuation. I would also suggest it is still better to be grammatically correct.

> b) whatever grammar-tweaks you make here, you should probably also duplicate in the Mac OS section?

Good point - I've done that.